src/Common/src/Interop/Windows/kernel32/Interop.CTL_CODE.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Data.Common;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        /// <summary>
+        /// <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/defining-i-o-control-codes">CTL_CODE</a> method.
+        /// </summary>
+        internal unsafe static uint CTL_CODE
+        (
+            /// <summary>
+            /// Identifies the device type. This value must match the value that is set in the DeviceType member of the driver's DEVICE_OBJECT structure.
```

Comment:
Arguments should be described via `/// <param name="deviceType"></param>`, etc in the block with the summary.